department-of-veterans-affairs/abd-vro
# MCP-1873 PDF Generator Error Reason Field

#MCP-1873 PDF Generator Error Reason Field 

## Description



- PDF Generator was still returning a 200 even though we got a `status: ERROR`
- PDF Generator now adds a `reason` field in addition to the `status: ERROR` to give a better description of what went wrong

### Jira Tickets



- [MCP-1873](https://amida.atlassian.net/browse/MCP-1873)

## How to test this PR

- Turn off Redis container and generate a PDF. Status should be `500` and you should see a `reason`
